python_apps/pypo/liquidsoap/generate_liquidsoap_cfg.py

```diff
@@ -39,10 +39,11 @@ def run():
             generate_liquidsoap_config(ss)
             successful = True
         except Exception, e:
+            print "Unable to connect to the Airtime server."
+            logging.error(str(e))
+            logging.error("traceback: %s", traceback.format_exc())
             if attempts == max_attempts:
-                print "Unable to connect to the Airtime server."
-                logging.error(str(e))
-                logging.error("traceback: %s", traceback.format_exc())
```

Comment:
This old logic is dumb because the program will print absolutely nothing and just hang for 30 seconds until it finishes trying 10 times. I just shuffled the code around to at least print something letting you know what the problem was without you having to wait 30 seconds...